### ISSUE TITLE
Image tagged as stable doesn't working?

### ISSUE THREAD

**Issue by pwFoo** [rancher/rancher#5078]
I installed the stable tagged image after some problems with latest, but some basics doesn't work anymore. Also a can't add any custom hosts.

Stable tagged images should be stable and working, but it seems latest dev is much more stable at the moment?

**Comment by futuretec** [comment]
Could you be more specific? Which basics don't seem to work anymore?

Which Docker version are you using for your custom hosts and can they reach the Rancher server + ports accessible?

**Comment by pwFoo** [comment]
It wasn't possible to add hosts with rancher/server:stable. Same hosts / setup (RancherOS, Hosts unmodified between tests) works fine with dev / latest.

I tested the "stable" images because of some LB problems (#4985)

Back to latest images add hosts work again, but LB at the host with rancher server running stucks in "initializing" again. Other hosts (without rancher server) work fine.

**Comment by deniseschannon** [comment]
@pwFoo It would help if you could provide more details of your setup. 

What version of Rancher OS are you running for your set up? Where are they setup? What kind of environment are you using?

**Comment by pwFoo** [comment]
RancherOS 0.4.5, bare metal install at 
1. vmware ( + Rancher Server)
1. kvm
1. pcengines apu, dynIp, for testing

Tested different setups. Host rancher port 8080 is by reachable
* public ip
* nat from private to public ip
* reverse proxy

I experienced much more problems with stable compared to latest.

I reinstalled it with latest and all hosts added successfully and most ist working. Only LB at host with rancher server stucks with "initialization". LBs at other two hosts (KVM and pcengines apu behind dynip and nat) works fine.
*
